requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+-r requirements.txt
+mock
+pep8==1.6.2
```

Comment:
> Note that in `.travis.yml`, we're now using this `requirements-dev.txt` so local and Travis will have the same versions.

Ah, sorry, missed that we have full control over the environment for Travis.

> but consider that 1.7.0 might introduce breaking changes again

That's true. I guess requiring 1.6.2 is fine then.